python/lsst/pex/config/config.py

```diff
@@ -1250,6 +1254,10 @@ def loadFromString(self, code, root="config", filename=None):
         with RecordingImporter() as importer:
             globals = {"__file__": filename}
             local = {root: self}
+            if extraLocals is not None:
+                # remote any extraLocal with the name of root
```

Comment:
This seems like it should be an error instead; don't want the caller to think they got away with passing `root` just to get mysterious behavior.

Reply:
Remove instead of remote?